api/src/routing.js

```diff
@@ -275,9 +279,9 @@ app.get(
   (req, res) => res.sendFile(path.join(resources.webappPath, 'appcache-upgrade.html'))
 );
 
-app.all('/+medic(/*)?', (req, res, next) => {
+app.all('/medic{/{*any}}', (req, res, next) => {
```

Comment:
All these changes are due to https://expressjs.com/en/guide/migrating-5.html#path-syntax